ctfd.ini

```diff
@@ -0,0 +1,35 @@
+# UWSGI Configuration File
+# Install uwsgi (sudo apt-get install uwsgi), copy this file to
+# /etc/uwsgi/apps-available and then link it in /etc/uwsgi/apps-enabled
+# Only two lines below (commented) need to be changed for your config.
+# Then, you can use something like the following in your nginx config:
+#        location = /ctf { rewrite ^ /ctf/; }
+#        location /ctf {
+#        	include uwsgi_params;
+#        	uwsgi_pass unix:/run/uwsgi/app/ctfd/socket;
+#        }
+# The first line above can be removed if you're using / as SCRIPT_ROOT
+# After that, just reboot nginx and uwsgi with service and it should work
+[uwsgi]
+plugin = python
+module = wsgi
+
+master = true
+processes = 1
+threads = 1
+
+vacuum = true
+
+# Where you've put CTFD
+chdir = /var/www/ctfd/
+# Change /ctf to whatever SCRIPT_ROOT you want
+mount = /ctf=wsgi.py
```

Comment:
It's mostly meant as an example (not just to be blindly copied/pasted), but yes I can change